Core fixes

kouroshHakha opened the pull request.

pkerichang commented:
Technically openmdao is an optional dependency, I can merge once that is changed.

felixonmars commented:
Updated accordingly.

Commit:
1. Updated meas_cell/sim_cell: with pause flag you can view results after the script runs
2. fixed bugs in core_v2.TBM/MM
3. view_name now should not be part of the yaml file, when running sim_cell or meas_cell one can pass -x flag to force post-layout view simulation, view_sim_list still is supported.

Commit:
Merge branch 'master' of github.com:ucb-art/BAG_framework into core_fixes

Commit:
1. added import_sch_cellview function
2. added support for hierarchical wrappers using dut_params convention

Commit:
fixed a bug in measurementManager v2: extract was not a parameter in measure

Commit:
fixed a bug in measurementManager v2: extract was not a parameter in measure

Commit:
fixed bugs in core_v2 and simulate_cell

Commit:
refactored absolute dependencies to relative

curtisma commented:
Should this be merged?

kouroshHakha commented:
merged!

curtisma commented:
Excellent!  Thanks!
